File: URI-iniciante-Java/1099.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();

        for (int i = 0; i < n; i++) {
            int x = sc.nextInt();
            int y = sc.nextInt();
            int count = 0;
            if (x > y) {
                int temp = x;
                x = y;
                y = temp;
            }
            for (int j = (x + 1); j < y; j++) {
                if (j % 2 != 0) {
                    count += j;
                }
            }
            System.out.println(count);
        }
    }
}

File: Beginner/1099.py
n = int(input())

for i in range(0, n, 1):
    x = int(input())
    y = int(input())
    count: int = 0
    if x > y:
        temp = x
        x = y
        y = temp
    for j in range(x+1,y,1):
        if j % 2 == 0:
            count += j

else:
    print(count)

